docs/tools/aleph-client/documentation.md

```diff
@@ -0,0 +1,942 @@
+# Aleph CLI
+
+**Usage**:
+
+```console
+$ aleph aleph [OPTIONS] COMMAND [ARGS]...
+```
+
+**Options**:
+
+* `--install-completion`: Install completion for the current shell.
+* `--show-completion`: Show completion for the current shell, to copy it or customize the installation.
+* `--help`: Show this message and exit.
+
+**Commands**:
+
+* `about`: Display the informations of Aleph CLI
+* `account`: Manage account
+* `aggregate`: Manage aggregate messages on aleph.im
+* `domain`: Manage custom Domain (dns) on aleph.im
+* `file`: File uploading and pinning on IPFS and...
+* `instance`: Manage instances (VMs) on aleph.im network
+* `message`: Post, amend, watch and forget messages on...
+* `node`: Get node info on aleph.im network
+* `program`: Upload and update programs on aleph.im VM
+
+## `about`
+
+Display the informations of Aleph CLI
+
+**Usage**:
+
+```console
+$ aleph about [OPTIONS] COMMAND [ARGS]...
+```
+
+**Options**:
+
+* `--help`: Show this message and exit.
+
+**Commands**:
+
+* `version`
+
+### `about version`
+
+**Usage**:
+
+```console
+$ aleph about version [OPTIONS]
+```
+
+**Options**:
+
+* `--help`: Show this message and exit.
+
+## `account`
+
+Manage account
+
+**Usage**:
+
+```console
+$ aleph account [OPTIONS] COMMAND [ARGS]...
+```
+
+**Options**:
+
+* `--help`: Show this message and exit.
+
+**Commands**:
+
+* `address`: Display your public address.
+* `balance`
```

Comment:
Is it normal that there is no description for this?

Reply:
Okay, the name should be pretty self-explanatory, but I think it would be better to have a little description everywhere :)

Reply:
Yep, will update in the aleph CLI accordingly